src/copa_modules/_types/data_config.py

```diff
@@ -25,5 +25,6 @@ class data_config(TypedDict):
     output_dir: str
     output_file_name: str
     test_rows: int
+    test_mode: bool
```

Comment:
cument this file.

Reply:
It's a type definition that is used in both the `data_processor` class and `main.py`. To remove it, I would have to make changes in data_processor class.
Should I do it?